dataland-backend/src/main/kotlin/org/dataland/datalandbackend/utils/LogMessageBuilder.kt

```diff
@@ -9,7 +9,8 @@ import org.springframework.stereotype.Component
 
 @Component("LogMessageBuilder")
 class LogMessageBuilder {
-    val accessDeniedExceptionMessage = "You are trying to access a unreviewed dataset"
+    val accessDeniedExceptionMessage = "You are trying to access a unreviewed dataset."
```

Comment:
Ich habe glaube ich diese message damals selbst verfasst.
Jetzt im Nachhinein finde ich, dass die message `accessDeniedExceptionMessage` sehr allgemein klingt, dann aber einen expliziten Fall beschreibt.
Daher würde ich die Variable nach dem spezifischen Fall benennen, der ja in der Nachricht dann auch beschrieben wird.

Reply:
Hast du geändert und passt jetzt erstmal.